### ISSUE TITLE
Amina double triggering

### ISSUE THREAD

**Issue by Zerothmaxima** [mtgred/netrunner#4745]
tl;dr: DOOF run on Sportsmeta with  Gatekeepr as outer ice and Eli as inner ice. Leela broke 0str gatekeeper w/ amina, lost a cred, then broke Eli with Makler, and amina triggers again. Put arrows next to both occurrences and credit pool reflected that it actually happened. 

warplord spends and pays 1 to play Diversion of Funds.
warplord approaches Gatekeeper protecting HQ at position 1.
0thmxma has no further action.
warplord encounters Gatekeeper protecting HQ at position 1.
warplord uses Amina to break all 2 subroutines on Gatekeeper.
0thmxma has no further action.
warplord uses Amina to make the Corp lose 1 .  <-----------------------
warplord passes Gatekeeper protecting HQ at position 1.
warplord approaches Eli 1.0 protecting HQ at position 0.
0thmxma has no further action.
warplord encounters Eli 1.0 protecting HQ at position 0.
0thmxma has no further action.
warplord increases the strength of Makler to 4 and break all 2 subroutines on Eli 1.0.
warplord uses Amina to make the Corp lose 1 .   <-----------------------
warplord passes Eli 1.0 protecting HQ at position 0.
warplord uses Makler to gain 1 .
warplord approaches HQ.
0thmxma has no further action.
warplord chooses to use the replacement effect from Diversion of Funds.
warplord uses Diversion of Funds to force the Corp to lose 5, and gain 5.

**Comment by NoahTheDuke** [comment]
Thanks for the report! This will be fixed in the next update!
